check_dirty.py

```diff
@@ -0,0 +1,18 @@
+"""
+Checks if a repo is dirty.
+"""
+import argparse
+import sys
+from git import Repo
+
+def check_if_dirty(repository):
+	repo = Repo(args.repo_path)
+	if repo.is_dirty():
```

Comment:
Also, can you switch to 4 spaces indentation? That's what we seem to use most eventhough I don't think it's a defined standard anywhere.